### ISSUE TITLE
Onboard Shannon to NRRD website project team

### ISSUE THREAD

**Issue by jennmalcolm** [18F/doi-extractives-data#2778]
In order to get Shannon productively contributing to the NRRD website team, @mentastc  should help Shannon complete a prescribed set of tasks that will bring them up to speed.

# Directions

**Chris:** If you can’t complete any of the items on your checklist personally, you are responsible for ensuring that someone with the correct access completes that item.

**Shannon:** Take judicious notes on what about this onboarding process or the NRRD website is confusing or frustrating. If you notice a problem (especially with things like documentation), you are more than welcome to fix it! At the very least, please share this information with your buddy so we can make the team/platform better. You can also file issues and pull requests as soon as you feel comfortable doing so.

## Learn about the project background and how the team works

- [x] Read the [Product Framing](https://github.com/18F/doi-extractives-data/wiki/product-framing) document

- [x] Read about the current status of the project on the [homepage of the wiki](https://github.com/18F/doi-extractives-data/wiki/home)

- [x] `Chris`: Add `Shannon` to the [project team](https://github.com/18F/doi-extractives-data/wiki/Joining-the-Natural-Resources-Revenue-Data-project-team#current-project-team) list

- [x] Read the [Basics for making distributed work work](https://github.com/18F/doi-extractives-data/wiki/Basics-for-making-distributed-work-work)

- [x] Meet with the Product Manager for an introduction to project routines, expectations, and rituals
  - [x] Discuss [NRRD project meetings](https://github.com/18F/doi-extractives-data/wiki/Joining-the-Natural-Resources-Revenue-Data-project-team#meetings): what they are, how they are run, and how the new person is expected to participate at each
  - [x] Review the most recent [NRRD project roadmap](https://app.mural.ly/t/gsa6/m/gsa6/1517848877792/f45f99f44093ff10a347988b4fbabff36221e315) to get a high-level view of recently-completed, in-progress, and upcoming features

## Get access to tools and services

- [x] `Chris`: **Ask the 18F team PM to invite `Shannon` to the #nrrd-partners slack channel**
  - [x] `Shannon`: Accept the slack invitation via email,  and set up your Slack account with 2-factor authentication. Set up your profile with your name, location, and any other information you wish to provide.

- [x] **Create a GitHub account that you can use for government work**  
GitHub is a platform for collaborative web development. The NRRD website’s repository is currently housed under 18F’s GitHub organization, but in the future it will be transferred to ONRR. While it’s under 18F, accounts must follow 18F’s GitHub profile guidelines in order for team members to be added to it.
  - [x] Follow [18F’s setup instructions](https://handbook.18f.gov/github/#setup) to create a GitHub account, or link an existing account to your government work. 
  - [x] Let the Product Manager know when your account has been created, and what your username is. Ask them to add you as a contributor to the project repository so that you can make contributions, and to add you to the DOI GitHub organization.
  - [x] Subscribe to the [doi-extractives-data repository](https://github.com/18F/doi-extractives-data) (through the [GitHub watch feature](https://help.github.com/articles/watching-and-unwatching-repositories/)) to get notifications of project activity.

- [x] **Log in to [Waffle](https://waffle.io/) with your GitHub account so that you can view the project planning board**  
Select the “Public Repos Only” option when prompted to log in. We use Waffle as a project management tool to track tasks. Waffle connects to GitHub and displays the issues from the the project’s repository in columns that show its progress—Icebox, Product Backlog, Sprint Backlog, In Progress, Awaiting Review, or Done.

## Design checklist

- [x] **Read the design principles for the NRRD site**  
On the front page of the [style guide](https://revenuedata.doi.gov/styleguide/)

- [x] **Learn about our user research practice**  
Our research plans and findings are documented in the open as part of our GitHub repository. You can find the research at any time by changing the branch in the repo to the “research” branch.
  - [x] Read about our NRRD's [Research norms and processes](https://github.com/18F/doi-extractives-data/wiki/Research-norms-and-processes)
  - [x] Read through the research work from a few recent sprints. Each sprint has its own folder that contains a plan and the results:
    * [Research and findings from sprint 14](https://github.com/18F/doi-extractives-data/tree/research/research/14_sprint-inquisitive-impala)
    * [Research and findings from sprint 13](https://github.com/18F/doi-extractives-data/tree/research/research/13_sprint-preposterous-puffin)
    * Read through the rest at your own pace  

  You can find everything together in the branch index: https://github.com/18F/doi-extractives-data/tree/research  

- [x] **Read about [how to design with NRRD data](https://github.com/18F/doi-extractives-data/wiki/Little-rules-for-designing-with-data)**  

- [x] **Review the [NRRD design system](https://revenuedata.doi.gov/styleguide/index.html) and style guide**  
A collection of interaction and design patterns used across the site and the style rules that determine the site’s visual identity. 

- [x] **Review the [content guide](https://github.com/18F/doi-extractives-data/wiki/Content-guide)** to get a sense of how to write and talk about NRRD content

- [x] **If you are new to GitHub, learn about how it works, and how to make it work for you**  
GitHub is traditionally a development tool, not a design tool. It may take some time and experimenting to figure out how to work within its framework as a designer. Some places to start:
  * **Understand the [GitHub workflow](https://guides.github.com/introduction/flow/) that code-based changes to the website go through.**  
    Even if you won’t be contributing code changes, it helps to share the same vocabulary as your team.  
  * **Learn about [GitHub issues](https://guides.github.com/features/issues/)**  
    Issues are how the team tracks tasks to work on or bugs to fix.
  * **Understand the basics of writing in [Markdown](https://guides.github.com/features/mastering-markdown/)**  
    Markdown is a lightweight way of creating formatting for text. Using certain characters tells the text to render as headlines, bullets, etc. It will be helpful to know the basics of Markdown because GitHub uses it as the default syntax for writing issues (although they have recently implemented a WYSIWYG toolbar that helps!)

- [x] **Learn about [what you need to know about doing design and research in the federal government](https://github.com/18F/doi-extractives-data/wiki/What-you-need-to-know-about-doing-design-and-research-in-the-federal-government)**  

- [x] **Get access to Adobe design tools so that you can access the design files if you need them**  
The visual assets and layouts for the site were originally designed by the 18F team using a tool called Sketch, which is only compatible with Mac operating systems. The NRRD team uses Windows operating systems, and Adobe design products. Over the course of the  product’s transition from 18F to NRRD, we’ve been converting files to Adobe Illustrator as needed. For now, the native design files live in [Google Drive](https://drive.google.com/open?id=10-NddNUSmWZz8NEymAXj6kgZknMMOMmG), until a better approach can be identified.

- [x] **Join the UX Community of Practice email listserv** (optional)  
A mailing list of fellow UX designers across government. Useful for sharing resources and learning about challenges shared by other practitioners. Anyone with a government email address can join by sending a request to UX-COP-request@LISTSERV.GSA.GOV

**Comment by mcharg** [comment]
The link to the Product Framing document is broken.

**Comment by jenniferthibault** [comment]
Thanks for flagging this @mcharg ! When I created the template I didn't account for the links changing contexts. I'll fix them in this issue right now, and in the onboarding template right after.

**Comment by jenniferthibault** [comment]
Links in both places have been updated.

**Comment by mcharg** [comment]
The links are now working for me.

**Comment by mcharg** [comment]
I made it through the checklist.  There's a lot of helpful information in here!

Here's my feedback:
- I wouldn't separate out the design principles form the rest of the style guide.  I read through the whole style guide when I got to the design principles task, so then I thought the second item for the style guide was a duplicate.
- I would group the items around NRRD research and conducting research in the government, since a lot of the questions that popped up in my head when I was reading through the NRRD research part were answered when I got to the government research part.
